### PR TITLE
[Docs] Tune SQLTarget/SQLSource

### PR DESCRIPTION
**Pull request by george0st**
Repair **db_uri** -> **db_url**, add info helper for dialects, update samples

### CODE REVIEW

**Comment by george0st** [comment]
Hi @liranbg, please approve/review.

File: docs/data-prep/ingest-data-fs.md [hunk 1]
```diff
@@ -183,16 +183,17 @@ Tech Preview
 ```
 ```{admonition} Limitation
 Do not use SQL reserved words as entity names. See more details in [Keywords and Reserved Words](https://dev.mysql.com/doc/refman/8.0/en/keywords.html).
+Current supported version of SQLAlchemy see extra `pip install mlrun[sqlalchemy]`. See more details about [Dialects](https://docs.sqlalchemy.org/en/20/dialects/index.html)
```
Comment (jillnogold) [inline docs/data-prep/ingest-data-fs.md:186]: Do you mean - 
Check the current supported version of SQLAlchemy in [extras-requirements.txt](https://github.com/mlrun/mlrun/blob/development/extras-requirements.txt) and if necessary run `pip install mlrun[sqlalchemy]`  to update.

Comment (george0st) [reply docs/data-prep/ingest-data-fs.md:186]: Exactly

**Comment by gtopper** [comment]
Rebased and (hopefully) slightly improved in #5455.

**Pull request closed without merging**